client/my-sites/email/paths.js

```diff
@@ -169,6 +169,10 @@ export function emailManagementEdit(
 	);
 }
 
+export function inboxManagement() {
+	return `${ emailManagementPrefix }/inbox`;
+}
+
```

Comment:
The naming suggestion mentioned elsewhere also applies here.

Additionally you may need to consider more than one route since in Calypso most site-based routes have the site name in the url. In this scenario, this url options could be:
- `/email/inbox`
- `/email/inbox/<site-name>`

The first path doesn't have a site specified, and often presents the site-selector to the user.

You could represent the paths like this. 

```suggestion
export function emailManagementInbox( siteName = null ) {
	if ( siteName ) {
		return `${ emailManagementPrefix }/inbox/${ siteName }`;
	}

	return `${ emailManagementPrefix }/inbox`;
}

```